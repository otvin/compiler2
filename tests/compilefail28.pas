program compilefail28(output);
{cannot define a constant using itself}

const pi = 3.14;
	negpi = -negpi;

begin
	writeln(pi);
	writeln(negpi);
end.
