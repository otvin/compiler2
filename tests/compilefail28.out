Self-referencing constant definition 'negpi' in file:tests/compilefail28.pas line:5     column:14 
